pkgs/applications/radio/tlf/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, glib, perl, ncurses
+, hamlib, xmlrpc_c }:
+
+stdenv.mkDerivation rec {
+  pname = "tlf";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "0gniysjm8aq5anq0a0az31vd6h1vyg56bifc7rpf53lsh9hkzmgc";
+  };
+
+  nativeBuildInputs = [ autoconf automake pkgconfig perl ];
+  buildInputs = [ glib ncurses hamlib xmlrpc_c ];
+
+  preConfigure = "autoreconf --install";
+
+  configureFlags = [ "--enable-hamlib" "--enable-fldigi-xmlrpc" ];
+
+  installPhase = ''
+    mkdir -p $out/lib
+
+    # Hack around lack of libtinfo in NixOS
+    ln -s ${ncurses.out}/lib/libncursesw.so.6 $out/lib/libtinfo.so.5
+
```

Comment:
`make install` is the default. The link command could just go into `postInstall`?

Reply:
Hmm, sure, yeah. That worked. I tried something like that before but it didn't work. I'm not sure what phase I wrote but it was probably a non-existing one.